apps/v3-subgraph/schema.graphql

```diff
@@ -103,10 +107,26 @@ type Proposal @entity {
   sponsored: Boolean!
   "address that sponsored the proposal"
   sponsor: Bytes!
+  "transaction hash of the proposal sponsor"
+  proposalSponsorTxHash: Bytes!
+  "unix timestamp of when the proposal was sponsored"
+  proposalSponsorTxAt: BigInt!
   "indicates if the proposal is cancelled"
   cancelled: Boolean!
+  "transaction hash of the cancelled proposal"
+  proposalCancelTxHash: Bytes!
```

Comment:
same with these - all the proposal states that might not happen or don't happen until later in the proposal lifecycle will need to be nullable.

sponsor
cancelled
processed